var/spack/repos/builtin/packages/timemory/package.py

```diff
@@ -16,6 +16,7 @@ class Timemory(CMakePackage):
     maintainers = ['jrmadsen']
 
     version('develop', branch='master', submodules=True)
+    version('3.0.1', commit='ef638e1cde90275ce7c0e12fc4902c27bcbdeefd', submodules=True)
```

Comment:
If `3.0.1` is [released](https://github.com/NERSC/timemory/releases/tag/v3.0.1), we should use tag here.

Reply:
Done. But since submodules are required, it still cannot rely on the tar.gz GitHub archive.